Apps/APN/App.js

```diff
@@ -28,6 +29,14 @@ export default function App() {
     };
 
     prepare();
+    const listeners = [registerInAppEventListener()];
+
+    // Remove listeners once unmounted
+    return () => {
+      for (let listener of listeners) {
```

Comment:
Do we plan to add more listeners in the near future ? If not, then let's remove the array and for loop for just one listener.

Reply:
[Removed the loop](https://github.com/customerio/customerio-reactnative/pull/170/commits/0e91db20257e0a33585068919aaf99d6bfada0df)